.tx/config

```diff
@@ -2,15 +2,16 @@
 #
 # This file is part of Invenio.
 # Copyright (C) 2015-2018 CERN.
+# Copyright (C) 2023-2023 Graz University of Technology.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 [main]
 host = https://www.transifex.com
 
-[invenio.invenio-i18n-messages]
+[o:inveniosoftware:p:invenio:r:invenio-i18n-messages]
```

Comment:
is this a part of new client?

Reply:
yes. It is described in the migration workflow here: https://developers.transifex.com/docs/migrating-from-older-versions-of-the-client#add